src/config/routes.rb

```diff
@@ -33,4 +33,10 @@
       end
     end
   end
+
+  # Authentication Routes
+  match 'auth/:provider/callback', to: 'sessions#create', via: [:get, :post]
```

Comment:
`match` will be deprecated in the future. It is better to use `get` or `post`. Also, I think you should just `POST` to `sessions#create`.